core/src/cluster_info.rs

```diff
@@ -1427,15 +1436,14 @@ impl Node {
         let (_, repair) = bind();
         let (_, broadcast) = bind();
         let (_, retransmit) = bind();
-        let (storage_port, _) = bind();
 
         let info = ContactInfo::new(
             pubkey,
             SocketAddr::new(gossip_addr.ip(), gossip_port),
             SocketAddr::new(gossip_addr.ip(), tvu_port),
             SocketAddr::new(gossip_addr.ip(), tpu_port),
             SocketAddr::new(gossip_addr.ip(), tpu_via_blobs_port),
-            SocketAddr::new(gossip_addr.ip(), storage_port),
+            SocketAddr::new(gossip_addr.ip(), 0),
```

Comment:
Should that be within FULLNODE_PORT_RANGE?

Reply:
I think it should be '0.0.0.0:0' actually. A fullnode will not advertise a storage port. I have an update in my local, just not pushed yet.